index.md

```diff
@@ -52,3 +203,61 @@ which most in the community are members of.
 If you have technical questions around DNS flag day you can join
 [the DNS-operations mailing list](https://lists.dns-oarc.net/mailman/listinfo/dns-operations)
 and ask them there.
+
+Get in touch
+============
+
+For press & media inquiries please use media (at) dns-oarc.net and please put
+“DNS Flag Day" in the email subject line.
+
+- Web https://dnsflagday.net/
+- Twitter https://twitter.com/dnsflagday
+- Announcements: https://lists.dns-oarc.net/mailman/listinfo/dns-announce
+- Discussion: https://lists.dns-oarc.net/mailman/listinfo/dns-operations
+
+FAQ
+===
+
+- Q: Is DNS over UDP dead?
+
+  A: No, UDP will still be the main means of transportation.
```

Comment:
No. DNS over UDP is massively scaleable, very resource-efficient and fault-tolerant. UDP will remain the primary transport protocol for DNS. We just need to ensure that DNS over TCP is available everywhere when required, so that all implementers and operators can rely on its availability.